plan.md

```diff
@@ -1,9 +1,282 @@
+organisation and the learning culture we want to encourage.
+
+**Externally**, we have identified a clear need for user-centric technical training
+both based on requests from our immediately local community (particularly in the
+social good sector) and the engagement with our [existing online tutorials](https://github.com/dwyl/the-book/issues/69) and
+as evidenced by the number of training platforms and companies appearing weekly.
+
+#### How?
+
+We will use **our own products** across all of our training, demonstrating that
+we have successfully implemented what we are teaching consistently.
+
+As we do now, we will continue to evaluate each part our product and services
+cycle as a potential candidate for a learning resource and workshop.
+
+##### Workshops
+Workshops will be run *internally* in QuietSpace to share our knowledge with our
+team and with [QuietSpace](#co-working) members as well as for our [services
+clients](#services).
+
+They will be captured on video and edited so they can be shared with our remote
+community.
+
+Current workshop topics:
++ HTML/CSS for designers
++ User testing
++ [New stack](https://github.com/dwyl/technology-stack)
++ Automated testing
++ Pragmatic Agile development
+
+> Each workshop will have a defined structure and proposed audience.
+
+Initial incarnations of the 'HTML/CSS for designers', 'Pragmatic Agile
+Development' and 'Automated Testing' workshops have already been held and will
+be reviewed and iterated on.
+
+
+##### Video
+Whilst text-based tutorials have been our current way of disseminating knowledge,
+these are not great for visual learners and limit our audience to people who are
+comfortable using and navigating GitHub, which a lot of beginners are not.
+
+Expanding into video content will allow us to:
++ Establish a new channel of learning for our community (especially for visual learners)
++ Uphold our values of creating a learning community
++ Enhance our credibility as subject experts
++ Create in-bound links to our other content, products and services, in turn
+improving their SEO and exposure
+
+The **first step** is to create a **five minute introductory** video to each of
+our [learn tutorials](https://github.com/dwyl/video/issues/17#issuecomment-275573939)
+so that we can determine _what people are interested in learning more of_ and
+define which subjects to invest our time in developing content for.
+
+##### Metrics to track
++ Number of hours spent creating content
++ Number of click-throughs to other dwyl content
++ Number of click-throughs to dwyl websites
+
+
+### Co-working
+
+Running our own co-working space is how we create the best working environment
+for our London-based team members and for London-based organisations looking
+for a productive and creative workspace.
+
+> Currently, this is set up and known as QuietSpace.
+
+##### Why?
+Having full ownership of the place where we work means that we can feel like
```

Comment:
I was under the impression that part of the reason for no investors was to retain control of the company and part of the reason for running a coworking space was to retain control over the working environment. I was under the impression that control was an important factor.